src/Context/SHOGunClientContext.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import SHOGunClient from '@terrestris/shogun-util/dist/service/SHOGunClient';
```

Comment:
The naming here is confusing to me. This might be fixed in the `shogun-util` but i don't think this whole stuff should be called "Client".

Reply:
Fixed in 1da0ba7fb48188716994770cc234019c23aeb652.